civic.json

```diff
@@ -0,0 +1,50 @@
+
+{
+    "id": "",
```

Comment:
Should be a URL to the schema specification per [the DC gov website](http://open.dc.gov/civic.json/specification.html).

We can use

`"id": "https://raw.githubusercontent.com/DCgov/civic.json/master/schemas/schema-v1.json",`

from https://github.com/codefordc/civic.json/blob/master/schema.json#L3